tasks/gotest.py

```diff
@@ -943,3 +945,46 @@ def lint_go(
     only_modified_packages=False,
 ):
     raise Exit("This task is deprecated, please use `inv linter.go`", 1)
+
+
+@task
+def check_otel_dependencies(ctx):
+    with ctx.cd("test/otel"):
+        # Rename fixtures
+        ctx.run("mv dependencies.go.fake dependencies.go")
+        ctx.run("mv go.mod.fake go.mod")
+        ctx.run("mv go.sum.fake go.sum")
+
+        # Update dependencies to latest local version
+        ctx.run("go mod tidy")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should you check for errors in return here?